coral-common/src/main/java/com/linkedin/coral/common/transformers/SqlCallTransformers.java

```diff
@@ -18,7 +18,11 @@
 public class SqlCallTransformers {
   private final ImmutableList<SqlCallTransformer> sqlCallTransformers;
 
-  SqlCallTransformers(ImmutableList<SqlCallTransformer> sqlCallTransformers) {
+  public SqlCallTransformers() {
+    this.sqlCallTransformers = null;
+  }
+
+  public SqlCallTransformers(ImmutableList<SqlCallTransformer> sqlCallTransformers) {
```

Comment:
Why do we need this change? Can we just use the `of` method below? If we define a static factory method, we probably do not need a constructor.

Reply:
Good catch, it is not necessary and I will remove it.